app/CocoaPods/Supporting Files/Info.plist

```diff
@@ -64,7 +64,7 @@
 	<key>LSMinimumSystemVersion</key>
 	<string>$(MACOSX_DEPLOYMENT_TARGET)</string>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © 2014 CocoaPods. All rights reserved.</string>
+	<string>Copyright © 2015 CocoaPods. All rights reserved.</string>
```

Comment:
I don't think this is correct. https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies

The first year of publication should be retained in the copyright notice.

> The notice for visually perceptible copies should contain all three elements described below.
> ...
> 2. The year of first publication. ...
> ...